clientv3/mirror/syncer.go

```diff
@@ -78,7 +95,7 @@ func (s *syncer) SyncBase(ctx context.Context) (<-chan clientv3.GetResponse, cha
 			// If len(s.prefix) != 0, we will sync key-value space with given prefix.
 			// We then range from the prefix to the next prefix if exists. Or we will
 			// range from the prefix to the end if the next prefix does not exists.
-			opts = append(opts, clientv3.WithPrefix())
+			opts = append(opts, clientv3.WithRange(prefixEnd(s.prefix)))
```

Comment:
Oh. Ok. You mean the internal batch. I was thinking something else. Can you make the func in client public instead of copying it into mirror pkg?